lstchain/reco/disp.py

```diff
@@ -123,7 +133,7 @@ def disp_parameters_event(hillas_parameters, source_pos_x, source_pos_y):
     disp_container.sign = d[4]
     disp_container.miss = miss(disp_container.dx.value,
```

Comment:
If the result is multiplied with `u.m`, dx and dy must be `u.m`, right? So I would add replace `value` with `to_value(u.m)`, which will save us some headache once we switch to all parameters in telescope frame (degrees) because it will fail instead of silently attach the wrong unit.